server/main.lua

```diff
@@ -11,26 +11,60 @@ local availableJobs = {
 }
 
 -- Functions
-
 local function giveStarterItems()
     local Player = QBCore.Functions.GetPlayer(source)
     if not Player then return end
+    local citizenid = Player.PlayerData.citizenid
+    local firstname = Player.PlayerData.charinfo.firstname
+    local lastname = Player.PlayerData.charinfo.lastname
+    local gender = 'Male'
+    local birthdate = Player.PlayerData.charinfo.birthdate
+    local nationality  = Player.PlayerData.charinfo.nationality
+    if not Player.PlayerData.charinfo.gender == 1 then
+        gender = 'Female'
+    else
+        gender = 'Male'
+    end
     for _, v in pairs(QBCore.Shared.StarterItems) do
-        local info = {}
-        if v.item == "id_card" then
-            info.citizenid = Player.PlayerData.citizenid
-            info.firstname = Player.PlayerData.charinfo.firstname
-            info.lastname = Player.PlayerData.charinfo.lastname
-            info.birthdate = Player.PlayerData.charinfo.birthdate
-            info.gender = Player.PlayerData.charinfo.gender
-            info.nationality = Player.PlayerData.charinfo.nationality
-        elseif v.item == "driver_license" then
-            info.firstname = Player.PlayerData.charinfo.firstname
-            info.lastname = Player.PlayerData.charinfo.lastname
-            info.birthdate = Player.PlayerData.charinfo.birthdate
-            info.type = "Class C Driver License"
+        if v.item == 'id_card' then
+            local metadata = {
+                type = string.format('%s %s', firstname, lastname),
+                description = string.format('CID: %s  \nDOB: %s  \nGender: %s  \nNationality: %s ', citizenid, birthdate, gender, nationality),
```

Comment:
instead of using description metadata for this utilize, https://overextended.github.io/docs/ox_inventory/Functions/Client/displayMetadata

Will essentially do the same without us having to store a big chunk of extra text in the Database